hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -1,22 +1,58 @@
 package org.apache.mesos.hdfs.config;
 
 import com.google.inject.Singleton;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.conf.Configured;
 import org.apache.hadoop.fs.Path;
 
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 import java.util.Properties;
 
+/**
+ * Provides executor configurations for launching processes at the slave leveraging hadoop
+ * configurations.
+ */
 @Singleton
-public class SchedulerConf extends Configured {
+public class HdfsFrameworkConfig {
 
-  public SchedulerConf(Configuration conf) {
+  private Configuration hadoopConfig;
+
```

Comment:
We don't need the DEFAULT_HEAP_SIZE variable in my view. It is only used in one place. DEFAULT_HADOOP_HEAP_SIZE should be sufficient.